openshift/olm/knative-serving.catalogsource.yaml

```diff
@@ -428,6 +428,34 @@ data:
                 - deletecollection
                 - patch
                 - watch
+              - apiGroups:
+                - build.knative.dev
+                resources:
+                - builds
+                verbs:
+                - get
+                - list
+                - create
+                - update
+                - delete
+                - patch
+                - watch
+              - apiGroups:
+                - pipeline.knative.dev
```

Comment:
oh this will have to change at some point…
(but I need to change that upstream before :sweat: )